lib/schemaUtils.js

```diff
@@ -3610,6 +3610,15 @@ module.exports = {
       }
     });
 
+    // handle root path '/'
+    if (postmanPath === '/' && schemaPath === '/') {
```

Comment:
shouldn't this only be executed if the _.each doesn't run?

Reply:
yes, but `_.each` will not run in this case as there is path segment for root path.